modules/rostests/apitests/ntdll/implicit_tls.c

```diff
@@ -0,0 +1,144 @@
+/*
+ * PROJECT:     ReactOS API Tests
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     Test for support of implicit Thread Local Storage (TLS)
+ * PROGRAMMER:  Shane Fournier
+ */
```

Comment:
See https://reactos.org/wiki/Coding_Style#File_Structure
```suggestion
/*
 * PROJECT:     ReactOS API Tests
 * LICENSE:     LGPL-2.0-or-later (https://spdx.org/licenses/LGPL-2.0-or-later)
 * PURPOSE:     Tests for Implicit Thread Local Storage (TLS) support
 * COPYRIGHT:   Copyright 2025 Shane Fournier <???????????????>
 */
```
NOTES:
- Suggested license comes from the one used in all the other files of apitests/ntdll
- Email address to be filled.